src/components/MultiHopTrade/components/LimitOrderV2/LimitOrderConfirm.tsx

```diff
@@ -321,6 +320,13 @@ export const LimitOrderConfirm = () => {
 
         setLimitOrderTxComplete(quoteId)
 
+        if (result.data) {
+          setLimitOrderId({
+            id: quoteId,
+            orderId: result.data,
+          })
+        }
```

Comment:
We were never storing the final limit order id somewhere, that's needed to save in the action metadata, to retrieve the said limit order later when refetching and updating actions